build/webpack.dev.conf.js

```diff
@@ -61,8 +55,12 @@ const devWebpackConfig = merge(baseWebpackConfig, {
     new webpack.NoEmitOnErrorsPlugin(),
     // https://github.com/ampedandwired/html-webpack-plugin
     new HtmlWebpackPlugin({
+      title: 'ZWave  To MQTT',
```

Comment:
extra space snuck in there?
```suggestion
      title: 'ZWave To MQTT',
```